Web/starter_project_iconics/components/profile/profileLayout.tsx

```diff
@@ -0,0 +1,83 @@
+import Link from 'next/link'
+import React, { useState } from 'react'
+
+type ProfileProps = {
+  buttonText: string
+  element: React.ReactNode
+  text: string
+  innerText: string
+  currentPage: string
+}
+
+const ProfileLayout: React.FC<ProfileProps> = ({
+  buttonText,
+  element,
+  text,
+  innerText,
+  currentPage,
+}) => {
+  const [activeLink, setActiveLink] = useState(currentPage)
+
+  const handleLinkClick = (linkText: string) => {
+    setActiveLink(linkText)
+  }
+
+  return (
+    <div className="min-h-screen bg-white flex p-10 flex-col">
+      <div className="flex   py-4">
+        <h1 className="text-4xl font-bold text-gray-900">Profile</h1>
+      </div>
+      <div className="flex flex-col md:flex-row justify-center items-center bg-white pb-0">
+        <Link
+          href="/profile/my-info"
+          className={`text-lg font-semibold  text-gray-700 pl-0 pr-4 pt-2 pb-5 hover:text-primary ${
```

Comment:
Can you use the colors already defined in your tailwind.config file? If its not define you can add it too since you've used it other places